docs/request_signing.rst

```diff
@@ -0,0 +1,474 @@
+.. Copyright 2017 Covata Limited or its affiliates
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+======
+Signer
```

Comment:
It was intentional but again it might not be optimal - I wanted two sub-sections, and the examples I found did it this way. What do you think? Maybe it's better to split this into two pages instead.